pydicom/tests/test_util.py

```diff
@@ -366,13 +367,72 @@ def testImplVRcomma(self):
                                      is_implicit_VR=True)
         got = ds.ROIContourSequence[0].ContourSequence[0].ContourData
         config.reset_data_element_callback()
-
         expected = [2., 4., 8., 16.]
         if have_numpy and config.use_DS_numpy:
             assert numpy.allclose(expected, got)
         else:
             assert expected == got
 
+    def testNullValueForFixedVR(self):
+        # Wipe first Contour Data, mark as length 0
+        null_ds_bytes = self.ds_bytes.replace(b"\x08\x00\x00\x00",
+                                              b"\x00\x00\x00\x00")
+        contour_data = b"\x32\x2c\x34\x2c\x38\x2c\x31\x36"
+        null_ds_bytes = null_ds_bytes.replace(contour_data, b"")
+        fixer.fix_separator(b",", for_VRs=["DS", "IS"],
+                            process_unknown_VRs=False)
+        ds = filereader.read_dataset(BytesIO(null_ds_bytes),
+                                     is_little_endian=True,
+                                     is_implicit_VR=True)
+
+        assert ds.ROIContourSequence[0].ContourSequence[0].ContourData is None
+
+        config.reset_data_element_callback()
+
+    def testSpaceDelimiter(self):
```

Comment:
I know that the names often do not follow PEP8, but for new code we try to follow it, e.g. use `test_space_delimiter` etc.

Reply:
Makes sense, updated